docs/Configuration.md

```diff
@@ -75,17 +71,13 @@ In general it is best practice to not generate regular users by the AC Tool but
 
 Users can be configured in the same way as groups in the **user_config** section.
 
-```
-[User Id]
-     - name: user name (optional, if empty user id is taken)
-     - isMemberOf: comma separated list of groups (optional)
-     - description: (optional, description)
-     - path: (optional, path of the group in JCR)
-     - isSystemUser: the created user is a system user (AEM 6.1 and later) (default: false, optional)
-     - password: can be used to preset passwords (not allowed for system users) (optional)
-     - profileContent: allows to provide docview xml that will reset the profile to the given structure after each run (optional, since v1.8.2)
-     - preferencesContent: allows to provide docview xml that will reset the preferences node to the given structure after each run (optional, since v1.8.2)
-```
+property | comment | required
+--- | --- | ---
+name, description, path, isMemberOf | Work exactly as for groups | optional
+password | The PW for the user. Obviously this is stored in plain text and should only be used for test users | Required for non-system users, otherwise must not be set
+isSystemUser | Create users as system user (AEM 6.1 and later) | optional
+profileContent | Allows to provide [enhanced docview xml](https://jackrabbit.apache.org/filevault/apidocs/org/apache/jackrabbit/vault/util/DocViewProperty.html) that will reset the profile to the given structure after each run (optional, since v1.8.2) | optional
+preferencesContent | Allows to provide [enhanced docview xml](https://jackrabbit.apache.org/filevault/apidocs/org/apache/jackrabbit/vault/util/DocViewProperty.html) that will reset the preferences node to the given structure after each run (optional, since v1.8.2) | optional
```

Comment:
I would rather say since 1.9.0, because the enhanced docview is only supported since the upcoming version.